tensorrt_llm/_torch/attention_backend/sparse/minimax_m3/decode_wrapper/topk.py

```
@@ -0,0 +1,97 @@
+    init_blocks : always include blocks ``[0, init_blocks)``.
+    local_blocks : always include blocks
+        ``[valid - local_blocks, valid)`` (per row).
+    out : optional ``[total_q, num_kv_heads, topk]`` int32 buffer.
+
+    Returns
+    -------
+    ``[total_q, num_kv_heads, topk]`` int32, ascending block indices,
+    ``-1`` padded at the tail (the sparse FMHA kernel's
+    ``kv_block_indexes`` contract).
+    """
+    num_kv_heads, max_k_tiles, total_q = max_score_kv.shape
```

Ruff (0.15.20)

[warning] Unpacked variable `num_kv_heads` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)